doc/version-history.rst

```diff
@@ -4,6 +4,10 @@
 Version History
 ===============
 
+v18.1.0
```

Comment:
Needs to be `19.0.0` since we're adding a CSC.

Reply:
Updated